src/features/events/layout/EventLayout.tsx

```diff
@@ -27,11 +33,157 @@ const EventLayout: React.FC<SurveyLayoutProps> = ({
     (env) => new EventDataModel(env, parseInt(orgId), parseInt(eventId))
   );
 
+  const showTimeOnly = (value: Date) => {
+    return value.toLocaleTimeString('en-Us', {
+      hour: '2-digit',
+      hour12: false,
+      minute: '2-digit',
+      timeZone: 'UTC',
+    });
+  };
+
   return (
     <TabbedLayout
       baseHref={`/organize/${orgId}/projects/${campaignId}/events/${eventId}`}
       defaultTab="/"
-      subtitle={<Box alignItems="center" display="flex"></Box>}
+      subtitle={
+        <Box alignItems="center" display="flex">
+          <EventStatusChip state={model.state} />
+          <Box marginX={2}>
+            <ZUIFuture future={model.getData()}>
+              {(data) => {
+                const startDate = new Date(data.start_time);
+                const startTime = showTimeOnly(startDate);
+
+                const endDate = new Date(data.end_time);
+                const endTime = showTimeOnly(endDate);
+
+                const isToday =
+                  startDate.toDateString() === new Date().toDateString();
+                const endsOnSameDay =
+                  startDate.toDateString() === endDate.toDateString();
+                const endsOnToday =
+                  endDate.toDateString() === new Date().toDateString();
+
+                const labels: ZUIIconLabelProps[] = [];
+
+                if (startDate && endDate) {
```

Comment:
Nice work with this! Let's remember this logic, because I think we will want to extract it into a separate component in the future (the same type of text is used in many places).